src/openfluid/ui/waresdev/WareBuildInfoWidget.hpp

```diff
@@ -0,0 +1,86 @@
+/*
+
+  This file is part of OpenFLUID software
+  Copyright(c) 2007, INRA - Montpellier SupAgro
+
+
+ == GNU General Public License Usage ==
+
+  OpenFLUID is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OpenFLUID is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OpenFLUID. If not, see <http://www.gnu.org/licenses/>.
+
+
+ == Other Usage ==
+
+  Other Usage means a use of OpenFLUID that is inconsistent with the GPL
+  license, and requires a written agreement between You and INRA.
+  Licensees for Other Usage of OpenFLUID may use this file in accordance
+  with the terms contained in the written agreement between You and INRA.
+  
+*/
+
+/**
+  @file WareBuildInfoWidget.hpp
+
+  @author Jean-Christophe FABRE <jean-christophe.fabre@inra.fr>
+*/
+
+
+#ifndef __OPENFLUID_UIWARESDEV_WAREBUILDINFOWIDGET_HPP__
+#define __OPENFLUID_UIWARESDEV_WAREBUILDINFOWIDGET_HPP__
+
+
+#include <QWidget>
+
+#include <openfluid/dllexport.hpp>
+#include <openfluid/ui/common/ActionLabel.hpp>
+
+
+namespace Ui
+{
+  class WareBuildInfoWidget;
+}
+
+
+namespace openfluid { namespace ui { namespace waresdev {
+ 
+
+class OPENFLUID_API WareBuildInfoWidget : public QWidget
+{
+  Q_OBJECT;
+
+  private:
+
+    Ui::WareBuildInfoWidget *ui;
+
+  signals:
+    void settingsButtonClicked();
+
+    void jobsScrolled(bool Up);
+
+  public:
+
+    WareBuildInfoWidget(QWidget* Parent = nullptr);
+
+    ~WareBuildInfoWidget();
+
+    openfluid::ui::common::ActionLabel* getLabel(std::string LabelName);
```

Comment:
Prefer pass by ref arguments
According to OpenFLUID coding style, prefer avoid the `get` prefix when returning a pointer or a reference

Why not 3 "hard" methods? buildModeLabel(), installLabel() and aprallelLabel()